import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/helper/DocumentMapper.java

```diff
@@ -94,7 +99,12 @@ private ActionResult processDocumentMapping(DocumentMapping doc, ImportRequest i
 		switch (doc.getOperation()) {
 		case INSERT:
 			// For now, we pass the mapping. Find a better more efficient way.
-			result = this.destinationProcessor.insert(source, this.getFieldMappingObject(), this.getValueMappingObject());
+			if (destination != null) {
+				result = this.destinationProcessor.update(source, destination, this.getFieldMappingObject(), this.getValueMappingObject(),doc.isOverrideTitle(), importRequest);
```

Comment:
I think we have to avoid making a different operation form that one defined. We have INSERT, but we can perform update.
Try to set the correct operation in addDocumentMapping() method. When the dest. doc is present, the operation would be UPDATE.
Also keep the style similar to the AMP one (even if we don't have checkstyle here). E.g.: spaces after comma, the length of the line code, etc.